lib/tasks/example_data.rake

```diff
@@ -180,14 +156,24 @@ namespace :example_data do
                   course_min_age: course.min_age,
                   course_max_age: course.max_age,
                   itt_start_date: course.published_start_date,
-                  itt_end_date: course.end_date,
+                  itt_end_date: course.published_start_date + 9.months,
```

Comment:
Do we need a range of end dates? Well, a range of ITT durations really. Just wondering if them all being '9' is the right thing to do.